Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10
```

Comment:
Why not use the latest image? And alpine.
Would suggest:
`FROM node:alpine`

Reply:
This is bad practice use non-specific version in Dockerfiles

Reply:
Makes sense, maybe use the latest working one then (or where they will be the least compatibility problems down the road)